### PR TITLE
Testing react js

### PR DESCRIPTION
**Pull request by AadarshGhodasara**
Today's Task (28/01/2021)

- Create A Contact Us Module.
- Some Change In AddFramePhoto.js and AddClockPhoto.js file.

### CODE REVIEW

File: src/Pages/addPhoto/AddClockPhoto.js [hunk 1]
```diff
@@ -111,24 +109,31 @@ class AddPhoto extends React.Component {
...
                             id="del" 
                             style={{display:'none'}}
-                        />                        
-                        <img src={this.state.url || 'https://via.placeholder.com/400x300'} alt="Upload logo" className='img-tag' />
+                        />  
+                        <div className='show-img-tag'>
+                            {   this.state.url !== '' ? 
+                                <img src={this.state.url} alt="Upload logo" className='img-tag' />
+                                :
+                                <img src={'https://via.placeholder.com/400x300'} alt="Upload logo" className='default-img-tag' />
```
Comment (simformniki) [inline src/Pages/addPhoto/AddClockPhoto.js]: You can rewrite above condition as below:

` Upload logo`

File: src/Pages/addPhoto/AddClockPhoto.js [hunk 2]
```diff
@@ -25,7 +23,7 @@ class AddPhoto extends React.Component {
 
 
       uploadImg = (e) => {
-        
+        console.log('INNNNNNNN->-> -> ->');
         const selected = e.target.files[0];
         if(e.target.files?.[0] && types.includes(selected.type)){
```
Comment (simformniki) [inline src/Pages/addPhoto/AddClockPhoto.js]: rewrite above 2 lines as 

```
const selected = e.target.files?.[0];
        if(selected  && types.includes(selected.type)){
```

File: src/Pages/addPhoto/AddClockPhoto.js (hunk 1 above)
Comment (simformniki) [inline src/Pages/addPhoto/AddClockPhoto.js]: please update this condition as per 1st comment.

File: src/Pages/addPhoto/AddFramePhoto.js [hunk 3]
```diff
@@ -1,33 +1,32 @@
...
         };
     }
 
 
-      uploadImg = (e) => {
-        
+    uploadFrameImg = (e) => {
+        console.log('INNNNNNNN->');
         const selected = e.target.files[0];
         if(e.target.files?.[0] && types.includes(selected.type)){
```
Comment (simformniki) [inline src/Pages/addPhoto/AddFramePhoto.js]: rewrite above 2 lines as

```
const selected = e.target.files?.[0];
        if(selected  && types.includes(selected.type)){
```

File: src/Pages/addPhoto/AddFramePhoto.js [hunk 4]
```diff
@@ -57,30 +57,31 @@ class AddFramePhoto extends React.Component {
...
         }
-        console.log('===> ',this.state.url);
+        // console.log('===> ',this.state.url);
 
     }
 
     deleteImg = () => {
         console.log('del');
-        if( this.state.url !== '' ){
+        if( this.state.frameUrl !== '' ){
```
Comment (simformniki) [inline src/Pages/addPhoto/AddFramePhoto.js]: you can write above condition as:

`if( this.state.frameUrl){`

File: src/Pages/addPhoto/AddFramePhoto.js [hunk 5]
```diff
@@ -92,42 +93,49 @@ class AddFramePhoto extends React.Component {
...
+                            id="framedel" 
                             style={{display:'none'}}
-                        />                        
-                        <img src={this.state.url || 'https://via.placeholder.com/400x300'} alt="Upload logo" className='img-tag' />
+                        />  
+                        <div className='showFrame-img-tag'>
+                            {   this.state.frameUrl !== '' ? 
+                                <img src={this.state.frameUrl} alt="Upload logo" className='frameImg-tag' />
+                                :
+                                <img src={'https://via.placeholder.com/400x300'} alt="Upload logo" className='defaultFrame-img-tag' />
```
Comment (simformniki) [inline src/Pages/addPhoto/AddFramePhoto.js]: please update this condition as per one of my comments.

File: src/Pages/ContactUs/ContactUs.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+import './ContactUs.css';
+import { Form , Button } from 'react-bootstrap';
+import emailjs from 'emailjs-com';
+
+const ContactUs = () => {
+
+    const sendEmail = (e) => {
+        e.preventDefault();
+        emailjs.sendForm('service_c1haowq', 'template_hvxta7j', e.target, 'user_PkGvOVMKJrZ3K5udlcntm')
+          .then((result) => {
```
Comment (suketu-simform) [inline src/Pages/ContactUs/ContactUs.js]: Store api keys or any such keys in .env file

File: src/Pages/addPhoto/AddPhoto.js [hunk 7]
```diff
@@ -111,24 +110,31 @@ class AddPhoto extends React.Component {
                             onClick={this.deleteImg} 
                             id="del" 
                             style={{display:'none'}}
-                        />                        
-                        <img src={this.state.url || 'https://via.placeholder.com/400x300'} alt="Upload logo" className='img-tag' />
+                        />  
+                        <div className='show-img-tag'>
+                            {   this.state.url !== '' ? 
+                                <img src={this.state.url} alt="Upload logo" className='img-tag' />
```
Comment (suketu-simform) [inline src/Pages/addPhoto/AddPhoto.js]: de-structure this.state.url

**Pull request merged**